### DISCUSSION TITLE
I see models with no .json lately, how do I use them ? (only .pth, .index, and .npy files)

### DISCUSSION THREAD

**Discussion by FoxTrotte** [voicepaw/so-vits-svc-fork#708] [Q&A] [answered]
Hey everyone !

I started using this soft a couple of weeks ago, and as I was looking around for models to try it with, I started seeing more and more models being shared without a .json, and only a .pth, .index, and .npy file. The .pth being much smaller than usual.

Are these models currently unsupported, or are their any ways to usse them with so-vits-svc-fork ?

Thank you very much !!

**Comment by gvydon** [comment]
I have the same question too. Someone, please answer.

**Comment by sonphantrung** [comment]
Please give me an example of such model. Plus, do they have a tutorial on how to run that model?

**Reply by gvydon** [reply]
https://huggingface.co/QuickWick/Music-AI-Voices/tree/main
Most of these, for example. There is no tutorial, but there is a note that says "Created Using: SoftVC VITS Singing Voice Conversion (so vits svc 4.0) | Retrieval based Voice Conversion (RVC)".

**Reply by sonphantrung** [reply]
Strange, since models should always come with a config file (heck, all so-vits-svc 4.0 derived colabs/apps require a config file to run). And AFAICT, the files are in `.zip` format, did you extract the downloaded `.zip` files to see the content inside?

**Reply by Grownz** [reply]
Can you give an example? DL'ed one as an example and it had both .pth and .json files in it:
**
![grafik](https://github.com/voicepaw/so-vits-svc-fork/assets/33704684/13ce3b8e-8325-4d57-86a9-36f7586ca298)
**

**Reply by gvydon** [reply]
https://huggingface.co/QuickWick/Music-AI-Voices/tree/main/Dio%20Brando%20(From%20JoJo's%20Bizzare%20Adventure)%20(RVC)%2010k
https://huggingface.co/QuickWick/Music-AI-Voices/tree/main/Eminem(Slim%20Shady%20LP%20trained%20)%20800%20epoch%20Small%20Edition
https://huggingface.co/QuickWick/Music-AI-Voices/tree/main/Duke%20Nukem%20(RVC)%20300%20Epoch
There are a lot of them. Maybe there is a way to make a .json file yourself? I don't know. I've seen colabs that work with .zip files, but I am not sure if they will work with these kind of models. I think those colabs just exctract zip and use the same file structure as so-vits-svc-fork. Here's a code for one of them: https://i.imgur.com/5DzsnV0.png

**Reply by Grownz** [reply, marked as answer]
Ok, it seems that the combination of `.pth` and `.index` Files is for another voice changer: https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI

The files are mentioned in their FAQ:
![grafik](https://github.com/voicepaw/so-vits-svc-fork/assets/33704684/b12c661c-74fb-493b-ad98-1d0137d74919)

**Reply by gvydon** [reply]
Thank you, I'll try it!
